network/core-node/admin-guide/publishing-history-archives.mdx

```diff
@@ -197,54 +183,32 @@ put="cp {0} /mnt/xvdf/stellar-core-archive/node_001/{1}"
 mkdir="mkdir -p /mnt/xvdf/stellar-core-archive/node_001/{0}"
 ```
 
-</CodeExample>
-
-- Run new-hist to create the local archive
-
-`# sudo -u stellar stellar-core --conf /etc/stellar/stellar-core.cfg new-hist local`
-
-This command creates the History archive structure:
-
-<CodeExample>
+Next, you must run Stellar Core's `new-hist` command to create and initialize the local archive. (This is done as the `stellar` user.)
 
-```
-# tree -a /mnt/xvdf/stellar-core-archive/
-/mnt/xvdf/stellar-core-archive
-└── node_001
-    ├── history
-    │   └── 00
-    │       └── 00
-    │           └── 00
-    │               └── history-00000000.json
-    └── .well-known
-        └── stellar-history.json
-
-6 directories, 2 file
+```bash
+sudo -u stellar stellar-core --conf /etc/stellar/stellar-core.cfg new-hist local
 ```
 
-</CodeExample>
+Now you can start your Stellar Core instance again:
 
-- Start your Stellar Core instance (`systemctl start stellar-core`)
-- Allow your node to join the network and watch it start publishing a few checkpoints to the newly created archive
+```bash
+systemctl start stellar-core # modify this if not using systemctl
+```
 
-<CodeExample>
+As you allow your node to join the network again, you can watch it start publishing a few checkpoints to the newly created archive.
 
-```
+```log
 2019-04-25T12:30:43.275 GDUQJ [History INFO] Publishing 1 queued checkpoints [16895-16895]: Awaiting 0/0 prerequisites of: publish-000041ff
 ```
 
-</CodeExample>
-
-At this stage your validator is successfully publishing its history, which enables other users to join the network using your archive (although it won't allow them to `CATCHUP_COMPLETE=true` as the archive only has partial network history).
+At this stage your validator is successfully publishing its history, which enables other users to join the network using your archive (although it won't allow them to `CATCHUP_COMPLETE=true` using your node, as the archive only has partial network history).
```

Comment:
```suggestion
At this stage your validator is successfully publishing its history, which enables other users to join the network using your archive.
```

Reply:
I think the CATCHUP_COMPLETE line may be misleading (since that's a setting that replays history)